core/src/main/java/hudson/slaves/SlaveComputerUtil.java

```diff
@@ -0,0 +1,30 @@
+package hudson.slaves;
```

Comment:
No new classes in `hudson.*` please.

Please also use `agent` instead of `slave` (both package and class name).